app/build.gradle.kts

```diff
@@ -225,6 +225,10 @@ dependencies {
 
     implementation(libs.bundles.splitties)
 
+    // https://square.github.io/okhttp/changelogs/changelog/
```

Comment:
Use bundle instead

Reply:
It's a platform dependency, we'll extract anyway when migrating to kmp.

Reply:
ACK.